tools/pipelines/templates/include-publish-npm-package.yml

```diff
@@ -36,6 +36,24 @@ stages:
       pool: ${{ parameters.pool }}
       publishNonScopedPackages: ${{ parameters.publishNonScopedPackages }}
 
+- stage: publish_npm_internal_client_utils_package
+  dependsOn: build
+  displayName: Publish Client-Utils Internal Package
+  condition: and(succeeded(), eq(variables['testBuild'], false))
```

Comment:
it would be a bit nicer if this step handled publishing \@fluid-internal/client-utils for both test and non-test branches, that way we could get a little validation of this change via a test branch.

Reply:
I did the testing by running a build on a branch.

Reply:
could you link the run?

Reply:
https://dev.azure.com/fluidframework/internal/_build/results?buildId=190066&view=logs&j=d96bdcbf-8421-52a3-1dec-90d9b28316ad

Reply:
note to anyone else who might be looking at the logs--if you check the commit, you'll see the actual package publishing steps were commented out and replaced with an `echo` (hence why you won't see those package versions on npm or our internal feeds)